### DISCUSSION TITLE
Constant slow forward motion with Mixed Reality rig

### DISCUSSION THREAD

**Discussion by mediadog** [mozilla/hubs#3634] [General Questions & Discussion]
In a Hubs room, and only with Hubs, I get a constant slow forward motion.  Touching the left controller joystick back slightly stops it.  But as soon as I do any other motion and let the joystick center, it goes back to moving forward slowly again.

This does not happen anywhere else; I can bail out of Hubs and all is well in the MR homespace, and in SteamVR, etc. etc.  So it is something to do with Hubs.

Any way to force a controller recenter or recalibrate?  This makes Hubs unusable with this setup, which is an Acer MR HMD and controllers, Win10 Pro x64, Firefox 84.0.1 64-bit running on an Alienware 13 R2 with GTX 1060.  SteamVR 1.15.16, and all Windows updates current.

**Comment by mediadog** [comment]
More info:
If the controllers are off, and I start SteamVR and enter the Hubs room, there is no forward motion.  But as soon as I turn on the left controller, the forward motion starts.  Again this happens in no other VR application but Hubs.

**Comment by pattersonbl2** [comment]
Are you still having issues with this ?

**Reply by mediadog** [reply]
Yes this continued to be a problem, but I have not checked with any newer versions of Hubs in the last few months, but I did test with another WMR setup (Odyssey) at the time and it did not show the problem - so I have been using those.  I can check again with the Acer later this week.  Still haven't seen this problem anywhere else but Hubs.

**Reply by pattersonbl2** [reply]
I will keep this discussion open until you are able to work with newer versions of hubs. Let me know if you still have issues so I can open an issue for a dev team.
